lib/gitWorkflow.js

```diff
@@ -199,9 +193,11 @@ class GitWorkflow {
       const backupStash = await this.getBackupStash(ctx)
       const output = await this.execGit(['show', '--format=%b', `${backupStash}^3`])
       const untrackedDiff = typeof output === 'string' && output.trim() // remove empty lines from start of output
-      if (!untrackedDiff) return
-      await this.execGit([...gitApplyArgs], { input: `${untrackedDiff}\n` })
+      if (untrackedDiff) await this.execGit([...gitApplyArgs], { input: `${untrackedDiff}\n` })
     } catch (err) {} // eslint-disable-line no-empty
+
+    // If stashing resurrected deleted files, clean them out
+    await Promise.all(this.deletedFiles.map(file => removeFile(file)))
```

Comment:
Consider using a shorthand since the function `removeFile` accepts only one argument.